dashboard/views.py

```diff
@@ -53,10 +54,12 @@ def get(self, request, *args, **kargs):  # pylint: disable=unused-argument, no-s
 
         # create an instance of the client to query edX
         edx_client = EdxApi(user_social.extra_data, settings.EDXORG_BASE_URL)
-        # get an enrollments client for the student
+        # get enrollments for the student
         enrollments = get_student_enrollments(request.user, edx_client)
-        # get a certificates client for the student
+        # get certificates for the student
         certificates = get_student_certificates(request.user, edx_client)
+        # get current_grades for the student
+        current_grades = get_student_current_grades(request.user, edx_client)  # pylint: disable=unused-variable
```

Comment:
this has been put here on purpose: right now the data is not used, but it will be as soon as we work on the course statuses